console/backend/src/main/java/org/frankframework/console/controllers/FrankApiService.java

```diff
@@ -94,6 +94,11 @@ protected <T> T getProperty(String key, T defaultValue) {
 		return environment.getProperty(key, (Class<T>) defaultValue.getClass(), defaultValue);
 	}
 
+	@SuppressWarnings("unchecked")
+	protected <T> T getRequiredBooleanProperty(String key) {
+		return environment.getRequiredProperty(key, (Class<T>) boolean.class);
+	}
```

Comment:
Dit is niet handig, de console kan ook via Spring Boot draaien, dan gebruik je de application.properties file.